### DISCUSSION TITLE
Reset time when model is changed

### DISCUSSION THREAD

**Discussion by matbesancon** [ERGO-Code/HiGHS#920] [General]
I am solving a series of LPs, the difference between them is changes to the objective function only.
The solving time reported by HiGHS seems however not to be reset to 0 and therefore accumulates across iterations

**Comment by jajhall** [comment]
I'll think about changing this, but it's not an issue

**Comment by matbesancon** [comment]
We were working on a similar issue in SCIP, the moment where the solving-specific cache is reset is when the model parameters are reset (a coefficient change)

**Reply by jajhall** [reply]
HiGHS doesn't follow SCIP
